src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -8,6 +8,7 @@
 
 #include <antlr4-runtime.h>
 
+#include "absl/time/time.h"
```

Comment:
Can you (as a cleanup measure)
move as many of those includdes into the `.cpp` File as possible?
Most importantly this goes for the rather expensive ones from `engine/sparqlExpressions`.

Reply:
Moved those includes to `.cpp`. Some were also redundant.